src/js/toolbar.js

```diff
@@ -356,6 +356,7 @@ var Toolbar;
         showAndUpdateToolbar: function () {
             this.modifySelection();
             this.setToolbarButtonStates();
+            this.base.trigger('updateToolbar', {}, this.base.getFocusedElement());
```

Comment:
probably can't go before modifySelection because that would/could change the focusedElement, possibly. I know it has to go _after_ setToolbarButtonStates because of [something it is doing in there to reset buttons, I don't recall). but in the interest of not having to reposition a 2nd time it must go _before_ setToolbarPosition, because the size of the toolbar is calculated there, so the possibly-shorter-or-longer list of buttons needs to be already created by the time position calc happens. so I'm going to fallback to "yes, maybe, but can think of reasons why it is good to be right here in the middle"